create_aio_app/template/{{cookiecutter.project_name}}/requirements/documentation.in

```diff
@@ -0,0 +1,39 @@
+#
```

Comment:
it looks like input from `pip-compile`, are u sure that it's correct for `.in` file?

```
Sphinx==2.4.4
sphinx-autodoc-typehints==1.10.3
sphinxcontrib-websupport==1.2.0
doc8==0.8.0
```